duneanaobj/StandardRecord/SRGArTrack.h

```diff
@@ -0,0 +1,52 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRGArTrack.h
+// \brief   Reconstructed GAr track object
+// \author  F. Martinez <f.martinezlopez@qmul.ac.uk>
+// \date    Jan. 2023
+////////////////////////////////////////////////////////////////////////
+#ifndef DUNEANAOBJ_SRGARTRACK_H
+#define DUNEANAOBJ_SRGARTRACK_H
+
+#include "duneanaobj/StandardRecord/SRVector3D.h"
+#include "duneanaobj/StandardRecord/SRParticleTruth.h"
+
+namespace caf
+{
+  class SRGArTrack
+  {
+    public:
+      SRVector3D start;      ///< Track 3D start point
+      SRVector3D end;        ///< Track 3D end point
+      SRVector3D dir;        ///< Unit vector representing estimate of track direction *taken from start point*
+      SRVector3D enddir;     ///< Unit vector representing estimate of track direction *taken from endpoint*
+
+      // Track characteristics
+      float dEdx_F = -999.;  ///< dE/dx of track in forward fit
+      float dEdx_B = -999.;  ///< dE/dx of track in backward fit
+
+      float p_F = -999.;     ///< Reco momentum of track in forward fit
+      float p_B = -999.;     ///< Reco momentum of track in forward fit
+
+      float len_cm_F = -999.; ///< Length of track in forward fit
+      float len_cm_B = -999.; ///< Length of track in backward fit
+
+      int clusters_in_track = -999; ///< Total number of TPC clusters in track
+
+      int trk_id = -999;  ///< GArSoft trackId for track
```

Comment:
maybe `garsoft_trkid`?

Reply:
Sure, that seems like a much safer name choice. I've changed also the other id and assn branch names.